sections/04_study.md

```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
 probabilistic models with hand-developed inputs [@tag:Torracinta2016_sim]. We
 anticipate that these steps will be replaced by deep learning, which will infer
 features simply by its ability to optimize a complex model against data.
+
+### Structure determination and cryo-electron microscopy
+
+Recent advances in cryo-electron microscopy (cryo-EM) have allowed the
+structural characterization of proteins and macromolecular complexes at near
+atomic resolution [@doi:10.1016/j.cell.2015.03.049]. Complexes intractable to
+study by established approaches like X-ray crystallography and NMR have been
+solved using this method [@doi:10.1038/nature12822]. Structures are determined
```

Comment:
This paragraph reads well, but in the interest in keeping our massive review compact I think we can shorten it.  Can you condense the sentences starting with `Structures are determined` to emphasize what are the areas of need where deep learning is helpful?  My takeaway is that many images are produced rapidly and the analysis requires quick, automated methods to pick particles in real time.

Reply:
The suggestion was incorporated.  The first paragraph was revised for length.